.travis.yml

```diff
@@ -8,10 +8,10 @@ env:
         # Add the BINSTAR_TOKEN environment variable. This has been generated with "" then "travis encrypt BINSTAR_TOKEN=<token_id>"
         - secure: "WgworyUp3oy/lyP3Dd+1bxs13rAyjbGqwWLK1XQUUOkHV6uKt5ZHFBbKRtYDnrQLSx9aAu/DDbh47L1nE1XL4C7u+S018GL+w7UBwGHkWHxBkr0TlYeVGCtjCo0QXk//L+JJXfnQ/2voCeqUkypD/O28vUfreVaXBpjnRlAzYmysFoDYl86/+nxKcSRW9+twbKk5xWroyDF+Ed7lCmXvqGWKMcOu14Wp30C2hKi3lfPyV3AzQ/lSQmwDcoCVIMsUE4+AFZAhub3T6C/EWURgfMjoX861iG/9vgJgnLdn3TsVW3WN88b8b+AR3EstnBL4ZJcpS1BwPF1hj9ER5av4vGkn39ahKVhWvUJO04Z+L7Ag8uoSypQsSAjlm7g82nph7EcpjymZ2BW0kqShs4jLwz3iO7I/ipjUYwlgAXInsTRgBjwbdf8YOKJYVnoLZNiv5seRow6BiVPNEmiQ3z9GH2Xt1emVxYdCcMscvyqmTQeZrQKHsseo5BoENh4Rq86CKtP9Orx1tIWrZ1Ujbfx4MpXFBbkzB6wNb+KZ+xPv3uD7DrWdaurZ2Kgm6HBY8YYHyq286halRZ3ngPXHgVW7aRVr5yH/uw8hXGr64cboIXhqt7wgZaHFy1BXAVfyqsNAB3qkisQEQSXmB4R8l7gua+zVNaOwz7LjW0mzcCpQhz8="
 
-        # Add the GH_TOKEN environment variable. (This is the "conda-forge.github.io" token for the conda-forge-admin user.)
-        - secure: "hT3MZHjghHcNSTb0ILwP6aErtiYNN2mf+CKNaFeGBfDv9MjhNAWnph92Kx+hsPRKa3GWLEqUpLQo33ynvesj430Ok1qeQK9BrM2/3YHDxpWkvG+dHTg2+dmKlveYIVG+SUeU+g/vERVwryOYj73tMH/vC53u6PNcaTeuphzI3pKOcoxYmchzVLAJCMEXtI8MPXchFKIwRxkj2DcN1ApJm0ZeBnegvsH5VeCVkbDOEMwOylCFV7JcywN2pIh7aE7cWrm3HTEmC26E2O0k6K0h666Hpgh7Ia9wDViyt3GAopTUcR2+9fyeQ8RK3CJVbDnRPTgt92EKUpuMTvsEVOpdFFyJXcmEqyd89ZGGnvXQHQADIFJJBRyT0qb2I3haytV3u5UiWtBMwuGaURILxvUb33x+KDKTfeS0DtV6O18H+BGkFcj5UUVRICk0wob1B9NTFGlukeQRM5rzrz5uTA/SOVYHEyFca5EsHa0Iq0T8ec3bsNj7/qH9ecvcVJCE1pOGPrgb89pshfe5LB36mwgan1YwkvtHxlBtb/0WT8bwc0fCuO8Q720lprJ6wu3fF4rM4vHGn99fgjcu7LnjXixw9chg1IraUO5LLjJJDJYlbv4+Lu5+2jrt99cQSmpjIhYqNLbBd2UQvbNxj/wDsKun1knjRhkUqClvu8plk8crARc="
+        # Add the GH_TOKEN environment variable. (This is the "conda-forge.github.io" token for the conda-forge-manager user.)
+        - secure: "QVcAIw1ezgkfZ70nNQBsr03V9tzXv0Hb+mw759gNo1CBbydYTO5EutWUt/Nks6E12/SkX5I3CmlAHZC74dQXVzQVyU/+hdc0El2N51gkvJ02dua7701ClqOHbkZSPGxE2j/S7ERzJKdtH97N0EGILP9wnadQJs5kpF9MvB1oWOQmDmsGMZ6GGlj2FMfL/TmtNR4ADo8rxr/ylw1kX8anfrXCv7FShdviugpsjSpAQLShKbdeDPdhvrD8W0TlpUhgdgSU2kPXd6gfGKuety1EWh56MgarHg+unntvB8R2W0O07RgOiRa/Mplmeng18dUCJdT1i3rczB2BkeyeFeCihtXqa0KDx7bX4z1X+4YFrBNl/bnDeVijaBtvkK3T0lluVnmVWolRzO9Tc2LozzAUEhDih4QFL6QtxQP2NkRFY2xEcj0JWbBNvDCik7hakiuxBSSbzGr1/3zw2sRh35/6qB6Y7mE6a5Pk+MS1ExS3Z8YUp4Q9l589DeyBCNE6HQHwEdvzEPUW74Lm566lzhsI8D4IuVYZDZLi/HHPzfPav4WWj5jiwlLx9XOkfPgz6ezqLoThkKSp5JmpXxxWzrckOMzFOyUqqD9h7De80uJqM/SW926fCr4RrrExcimblNV0ro9SSHUUvw+ZEFM51P7Z2m6ATBMO7/lVQtpj4XCbs6M="
 
-        # Add the CIRCLE_TOKEN environment variable. (This will need re-generating in the near future using the conda-forge-admin user)
+        # Add the CIRCLE_TOKEN environment variable. (This will need re-generating in the near future using the conda-forge-manager user)
```

Comment:
I see. Sorry I can't help clarify this. Please just let me know what you decide.